### PR TITLE
Remove max_user_connections from config

### PR DESCRIPTION
**Pull request by EmmyMiao87**
Update max_user_connections by user property
```
set property `user` max_user_connections=100;
```

Change-Id: Ia2a14b7472a1e3d2065bd73ec2b5f0c081bfa5d3

### CODE REVIEW

File: docs/zh-CN/administrator-guide/config/user_property.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# User 配置项
+
+该文档主要介绍了 User 级别的相关配置项。User 级别的配置主要是生效范围为单个用户。每个用户都可以设置自己的 User property。相互不影响。
```
Comment (kangkaisen) [inline docs/zh-CN/administrator-guide/config/user_property.md:29]:
```suggestion
该文档主要介绍了 User 级别的相关配置项。User 级别的配置生效范围为单个用户。每个用户都可以设置自己的 User property。相互不影响。
```

File: docs/zh-CN/administrator-guide/config/user_property.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## 设置配置项
+
+FE 启动后，在MySQL 客户端，通过下面命令修改 User 的配置项：
+
+`SET PROPERTY [FOR 'user'] 'key' = 'value' [, 'key' = 'value']`
+
+具体语法可通过命令：`help set property;` 查询。
+
+User 级别的配置项只会生效给指定的用户，并不会影响其他用户的配置。
```
Comment (kangkaisen) [inline docs/zh-CN/administrator-guide/config/user_property.md:47]:
```suggestion
User 级别的配置项只会对指定用户生效，并不会影响其他用户的配置。
```

File: docs/zh-CN/administrator-guide/config/user_property.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+{
+    "title": "User 配置项",
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/config/user_property.md:3]:
```suggestion
    "title": "用户配置项",
```

**Review by morningman — DISMISSED** [review]
LGTM

File: docs/zh-CN/administrator-guide/config/user_property.md (hunk 2 above)
Comment (morningman) [reply docs/zh-CN/administrator-guide/config/user_property.md:47]: She insist ^^

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**